extensions/src/main/java/cm/aptoide/pt/extensions/StringExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package cm.aptoide.pt.extensions
+
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+object StringExtensions {
```

Comment:
What for do we need this object wrapping the extension function? 🤔

Reply:
my bad. fixed